nixos/modules/services/network-filesystems/ceph.nix

```diff
@@ -157,6 +159,22 @@ in
           A comma-separated list of subnets that will be used as cluster networks in the cluster.
         '';
       };
+
+      rgwMimeTypesFile = mkOption {
+        type = with types; nullOr str;
+        default = "${pkgs.mime-types}/etc/mime.types";
+        description = ''
+          Path to mime types used by radosgw.
+        '';
+      };
+    };
+
+    extraConfig = mkOption {
+      type = with types; attrsOf str;
```

Comment:
This is left over from me, there is an example for the OSD daemon, that probably should be done for each daemon type to not require people to actually know that (they might setup the OSDs differently or not with this module for example)

Reply:
I've added an example to the global extraConfig, additionally, as lejonet mentioned, there's one for the OSD section. I see that it might be unintuitive for users, but in return you get a bit of syntax checking and programmatic ease, which might be useful for some people.